src/types/spidLevel.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * SPID authentication level enum.
+ */
```

Comment:
for reference, add a link to [this page](http://www.agid.gov.it/agenda-digitale/infrastrutture-architetture/spid/percorso-attuazione)